src/Essentials/src/Geolocation/Geolocation.ios.macos.cs

```diff
@@ -183,7 +185,7 @@ public void StopListeningForeground()
 				listener.ErrorHandler = null;
 			}
 
-			listeningManager.Delegate = null;
+			listeningManager.WeakDelegate = null;
```

Comment:
I'm not sure setting the `WeakDelegate` is the proper thing to do here, but since the `Delegate` now is non-nullable, it can't be set to null anymore.

Reply:
@jfversluis the third...